pages/people_page.py

```diff
@@ -9,46 +9,49 @@
 class PeoplePage(Page):
 
     url = '/people'
-    admins = ['Tim Bortnik']
+    admins = ['Tim Bortnik', 'Yevhenii Udovychenko']
```

Comment:
@Bidonus  is it correct solution?

Reply:
maybe we should to do some function to add (or delete) user to admins list

Reply:
@Uarhodesian, I think you right, let's do it in next split?